AirportDepartures.playground/Contents.swift

```diff
@@ -29,9 +70,14 @@ import UIKit
 //: d. Make one of the flights have a `nil` terminal because it has not been decided yet.
 //:
 //: e. Stretch: Look at the API for [`DateComponents`](https://developer.apple.com/documentation/foundation/datecomponents?language=objc) for creating a specific time
-
-
-
+let flight93 = Flight(date: "12/25/20", terminal: "V" , flightStatus: FlightStatus.scheduled, destination: "Svalbard, Norway", name: "Flight 93", destinationDate: "12/26/20" )
+let flight111 = Flight(date: "12/24/20", terminal: nil, flightStatus: FlightStatus.enRoute, destination: "Rochester, New York", name: "Flight 111", destinationDate: "12/25/20")
+let flight13 = Flight(date: nil, terminal: nil, flightStatus: FlightStatus.canceled, destination: "Seattle, Washington", name: "Flight 13", destinationDate: nil)
+var flights: [Flight] = []
```

Comment:
This works but could have been `var flights:[Flight] = [flight93,flight111,flight13]`  and  skipped the next three.

Reply:
it kept saying something "extra arguments at positions #2, #3 in call"